opencti-platform/opencti-front/src/private/components/common/stix_core_objects/StixCoreObjectContentRoot.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FunctionComponent } from 'react';
+import StixCoreObjectContentHeader from '@components/common/stix_core_objects/StixCoreObjectContentHeader';
+import { Route, Routes, useLocation } from 'react-router-dom';
+
+import ContainerContent from '@components/common/containers/ContainerContent';
+import StixCoreObjectContent from '@components/common/stix_core_objects/StixCoreObjectContent';
+
+interface StixCoreObjectContentRootProps {
+  stixCoreObject: unknown;
+  isContainer?: boolean;
```

Comment:
can we know that the stixCoreObject is a container directly from stixCoreObject ?

Reply:
this boolean will disappear when content mapping is extended to all entities